content/en/docs/collector/internal-telemetry.md

```diff
@@ -272,3 +272,76 @@ The Collector logs the following internal events:
 - Data dropping due to invalid data stops.
 - A crash is detected, differentiated from a clean stop. Crash data is included
   if available.
+
+## Use internal telemetry to monitor the Collector
+
+This section recommends best practices for monitoring the Collector using its
+own telemetry.
+
+### Critical monitoring
+
+#### Data loss
+
+Use the rate of `otelcol_processor_dropped_spans > 0` and
+`otelcol_processor_dropped_metric_points > 0` to detect data loss. Depending on
+your project's requirements, select a narrow time window before alerting begins
+to avoid notifications for small losses that are within the desired reliability
+range and not considered outages.
+
+#### Low on CPU resources
+
+This depends on the CPU metrics available on the deployment, eg.:
+`kube_pod_container_resource_limits{resource="cpu", unit="core"}` for
+Kubernetes. Let's call it `available_cores` below. The idea here is to have an
+upper bound of the number of available cores, and the maximum expected ingestion
+rate considered safe, let's call it `safe_rate`, per core. This should trigger
+increase of resources/ instances (or raise an alert as appropriate) whenever
+`(actual_rate/available_cores) < safe_rate`.
+
+The `safe_rate` depends on the specific configuration being used. // TODO:
+Provide reference `safe_rate` for a few selected configurations.
```

Comment:
I made an attempt to edit this section and then reverted it to the original for two reasons. 

First, I don't think I have a good understanding of what it's trying to say, so my edits might be inaccurate. (Shouldn't the equation be `(actual_rate/available_cores) > safe_rate`?) 

Second, it's referring to a metric that is not emitted by the core Collector, so I am questioning whether this content even belongs here. Maybe we could swap it out for something about performance testing, using some of the `process` metrics compared with the `exporter_sent` metrics for a given time frame? 

@open-telemetry/collector-approvers, could you help?

Reply:
I would leave this out as well for the first version, since it's environment specific it feels less useful

Reply:
Removed in 878c850. :+1: